src/omv/ports/stm32/sensor.c

```diff
@@ -57,6 +57,17 @@ void ISC_SPI_IRQHandler(void) {
 }
 #endif // ISC_SPI
 
+#if defined(OMV_MDMA_CHANNEL_DCMI_0)
```

Comment:
The ISRs for this should never fire. So, this isn't needed. However, it should also be harmless to have.